fv3_cap.F90

```diff
@@ -0,0 +1,1691 @@
+    if (nbits < 1 .or. nbits > 31)  nbits=0  ! lossless compression (no quantization)
+
+    if(mype == 0) print *,'af nems config,quilting=',quilting,'calendar=', trim(calendar),' iau_offset=',iau_offset
+    if(mype == 0) print *,'af nems config,ideflate=',ideflate,'nbits=',nbits
+!
+    nfhout = 0 ; nfhmax_hf = 0 ; nfhout_hf = 0 ; nsout = 0
+    if ( quilting ) then
+      CALL ESMF_ConfigGetAttribute(config=CF,value=write_groups, &
+                                   label ='write_groups:',rc=rc)
+      if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
+!
+      CALL ESMF_ConfigGetAttribute(config=CF,value=wrttasks_per_group, &
+                                   label ='write_tasks_per_group:',rc=rc)
+      if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
+
+      CALL ESMF_ConfigGetAttribute(config=CF,value=app_domain, default="global", &
+                                   label ='app_domain:',rc=rc)
+      if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
+
+      CALL ESMF_ConfigGetAttribute(config=CF,value=isrcTermProcessing, default=0, &
+                                   label ='isrcTermProcessing:',rc=rc)
+      if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
+
+      if(mype == 0) print *,'af nems config,quilting=',quilting,'write_groups=', &
+        write_groups,wrttasks_per_group,'calendar=',trim(calendar),'calendar_type=',calendar_type, &
+        'isrcTermProcessing=', isrcTermProcessing
+!
+      CALL ESMF_ConfigGetAttribute(config=CF,value=num_files, &
+                                   label ='num_files:',rc=rc)
+      if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
+!
+      allocate(filename_base(num_files))
+      CALL ESMF_ConfigFindLabel(CF,'filename_base:',rc=RC)
+      do i=1,num_files
+        CALL ESMF_ConfigGetAttribute(config=CF,value=filename_base(i), rc=rc)
+        if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
+      enddo
+
+      allocate(output_file(num_files))
+      num_output_file = ESMF_ConfigGetLen(config=CF, label ='output_file:',rc=rc)
+      if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
+      if (num_files == num_output_file) then
+        CALL ESMF_ConfigGetAttribute(CF,valueList=output_file,label='output_file:', &
+             count=num_files, rc=RC)
+        if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
+        do i = 1, num_files
+          if(output_file(i) /= "netcdf" .and. output_file(i) /= "netcdf_parallel") then
+            write(0,*)"fv3_cap.F90: only netcdf and netcdf_parallel are allowed for multiple values of output_file"
```

Comment:
ERROR to ESMF log file as well - which will show a stack trace in the log and bring people to this line